### PR TITLE
Update metricflow-commands.md

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a callout to the mf commands doc where we’ve made changes to metrics an dyou want to have those changes reflected in the SL without rebuilding all the models

### CODE REVIEW

File: website/docs/docs/build/metricflow-commands.md [hunk 1]
```diff
@@ -84,6 +84,10 @@ You can use the `dbt sl` prefix before the command name to execute them in the d
 - [`tutorial`](#tutorial) &mdash; Dedicated MetricFlow tutorial to help get you started.
 -->
 
+:::tip Run dbt parse to reflect metric changes
+When you make changes to metrics, make sure you run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file is updated, reflecting your changes when you query metrics. Running `dbt parse` ensures you don't have to rebuild all the models.
```
Comment (nghi-ly) [inline website/docs/docs/build/metricflow-commands.md]:
```suggestion
When you make changes to metrics, make sure to run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file, reflecting your changes when querying metrics. By running `dbt parse`, you won't need to rebuild all the models.
```

File: website/snippets/_sl-test-and-query-metrics.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ Refer to the following steps to get started:
 2. Run a dbt command, such as `dbt parse`, `dbt run`, `dbt compile`, or `dbt build`. If you don't, you'll receive an error message that begins with: "ensure that you've ran an artifacts....".
 3. MetricFlow builds a semantic graph and generates a `semantic_manifest.json` file in dbt Cloud, which is stored in the `/target` directory. If using the Jaffle Shop example, run `dbt seed && dbt run` to ensure the required data is in your data platform before proceeding.
 
-:::info Run dbt parse when you make changes to metrics
-Any time you make changes to metrics, you need to run `dbt parse` at a minimum. This ensures the `semantic_manifest.json` file is updated and you can have your changes reflected when querying metrics.
-:::
+:::tip Run dbt parse to reflect metric changes
+When you make changes to metrics, make sure you run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file is updated, reflecting your changes when you query metrics. Running `dbt parse` ensures you don't have to rebuild all the models.
```
Comment (nghi-ly) [inline website/snippets/_sl-test-and-query-metrics.md]:
```suggestion
When you make changes to metrics, make sure to run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file, reflecting your changes when querying metrics. By running `dbt parse`, you won't need to rebuild all the models.
```

**Review by nghi-ly — APPROVED** [review]
looks good! left some minor suggs

File: website/docs/docs/build/metricflow-commands.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/build/metricflow-commands.md]: oo great catch!

**Pull request merged**